packages/flame/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
  - Fix render order of components and add tests
  - `isHud` renamed to `respectCamera`
  - Fix `HitboxCircle` when component is flipped
+ - `MoveAlongPathEffect` can now be absolute, and can auto-orient the object along the path
```

Comment:
Yeah, factory names sounded cool at first when there were only 4 effects or so, but as there are more and more of them, it's kinda hard to make them all follow the same pattern without sounding ridiculous.

In this particular case it's even worse because `MoveAlongPathEffect` is not a specialization of `MoveEffect` and should inherit from `Transform2DEffect` directly (which makes it ineligible for factory construction from `MoveEffect`), plus with the introduction of the `oriented=true` flag, it now performs both movement and rotation at the same time. So it feels unnatural to have it "inside" the MoveEffect.

Reply:
Fair enough!